Add a LESS file

ed1nh0 opened the pull request:
An alternative option to SCSS.
Plus it is reduced by its specificity and by default all the balloons are top positioned, thus there is no need to declare `[data-balloon-pos="up"]`.

ed1nh0 committed:
Add a LESS file

An alternative option to SCSS.

kazzkiq commented:
Will be taking a look at it tonight, thanks for the contribution.

kazzkiq commented:
@ed1nh0 Awesome job porting the project to LESS, however there were some recent changes on the original SASS file (related to browser compatibility issues) #16. In order to accept a LESS port there should be no compatibility differences between SASS and LESS files. Can you please update the LESS file with those changes?

ed1nh0 commented:
@kazzkiq Thank you! I'll do that.

ed1nh0 committed:
New LESS ported from SCSS

There was recent changes on the original SASS file so I've updated the LESS file. But I keep the specificity low and there is no need to declare `[data-balloon-pos="up"]` by default.

ed1nh0 committed:
Added files via upload

ed1nh0 commented:
@kazzkiq Done! The SCSS was successfully ported to LESS.

kazzkiq commented:
Thanks!
